src/coreclr/src/jit/importer.cpp

```diff
@@ -7560,6 +7560,11 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
         /* Get the call site sig */
         eeGetSig(pResolvedToken->token, pResolvedToken->tokenScope, pResolvedToken->tokenContext, &calliSig);
 
+        if (calliSig.callConv == CORINFO_CALLCONV_UNMANAGED)
```

Comment:
The VM side should be responsible for decoding the calling convention for the JIT into the a enum defined in corinfo.h (we may be able repurpose `CorInfoUnmanagedCallConv` for that).

Reply:
But that isn't how this specific API is defined. Is the suggestion we update the `CORINFO_SIG_INFO` data structure? This PR was to create the API so it can be consumed and block usage until full support can be added.

Reply:
> Update the `CORINFO_SIG_INFO` data structure

I think something like that will be required to get it fully supported.

> it can be consumed and block usage until full support can be added

Sounds good. Doing this in second step is fine.

Reply:
Oh wait. Shoot. I was being dense here. @BruceForstall alluded to a similar point I was missing. I can instead keep this validating in the VM side in the call to `eeGetSig()`. The VM knows that the JIT doesn't support this value so we can throw on that side and then when we add support we can then the JIT and VM in a unified fashion.